src/components/Form.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../App.css';
+
+const Form = () => (
+  <form className="Addbookform">
+    <input type="text" placeholder="book title" />
+    <input type="text" placeholder="author" />
+    <button type="button">Add Book</button>
```

Comment:
- Don't forget that this button type is "submit" 😉 So the form works fine when you come to implement the adding books feature. 😄